frontend/src/metabase/query_builder/containers/QuestionEmbedWidget.jsx

```diff
@@ -92,6 +92,10 @@ export default class QuestionEmbedWidget extends Component {
     isPublicLinksEnabled = MetabaseSettings.get("enable-public-sharing"),
     isEmbeddingEnabled = MetabaseSettings.get("enable-embedding"),
   }) {
+    if (question.isDataset()) {
+      return false;
+    }
```

Comment:
I'm not sure if we should only disable *Embed in other applications* but not public embed. But I'm following the repro, which only expects the button to not be visible.

Reply:
I am sorry for the late reply @WiNloSt. Tbh, I am also not sure to which extent do we want to deal with models and embedding. It seems like that is still being talked about, so I'm hoping @brunobergher could be of more help than I am.

Reply:
Embedding = Signed Embedding + Public Sharing = so both, hide the icon.

Reply:
> Embedding = Signed Embedding + Public Sharing = so both, hide the icon.

Thanks!